website/content/docs/provisioners/powershell.mdx

```diff
@@ -81,7 +83,7 @@ provisioner "powershell" {
   running on AWS, Azure, Google Compute, or OpenStack and would like to access
   the autogenerated password that Packer uses to connect to the instance via
   WinRM, you can use the `build` template engine to inject it using
-  ```{{ build `Password` }}```. In HCL templates, you can do the same thing by
+  `` {{ build `Password` }} ``. In HCL templates, you can do the same thing by
```

Comment:
This is strictly formatting, have confirmed these both render to `` {{ build `Password` }} `` (as seen on the [live page](https://www.packer.io/docs/provisioners/powershell)).